clang-tools-extra/test/clang-doc/test-path-abs.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: rm -rf %t && mkdir %t
+// RUN: cp "%s" "%t/test.cpp"
```

Comment:
Looks like I missed this. Please don’t copy the test file around. To be blunt this pattern has been brought up in multiple PRs and even in previous incarnations of this one. Use the same conventions we’ve insisted on in other tests here.

Reply:
Sorry, the reason why I kept using this pattern is because I copied the same test setups in the original test for clang-doc [here](https://github.com/llvm/llvm-project/blob/main/clang-tools-extra/test/clang-doc/single-file.cpp#L4) and [here](https://github.com/llvm/llvm-project/blob/main/clang-tools-extra/test/clang-doc/single-file-public.cpp#L4)